sdk/python/pulumi_digitalocean/tables.py

```diff
@@ -147,9 +142,6 @@
     "serviceSubnet": "service_subnet",
     "sha1Fingerprint": "sha1_fingerprint",
     "snapshotId": "snapshot_id",
-    "spacesAccessId": "spaces_access_id",
-    "spacesEndpoint": "spaces_endpoint",
-    "spacesSecretKey": "spaces_secret_key",
```

Comment:
@pgavlin Just to check - are these removals expected as a result of the codegen changes?

Reply:
There shouldn't be any codegen changes with this payload unless DO was on an older version of the bridge than I was aware of.

Reply:
*SDK codegen changes

Reply:
Okay, it looks like this has something to do with provider configuration. All of the removed properties are top-level entities in the config.